source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigParseResult.mm

```diff
@@ -12,10 +12,9 @@ @implementation ACOHostConfigParseResult
 - (instancetype)init:(ACOHostConfig *)config errors:(NSArray<NSError *> *)errors
 {
     self = [super init];
-    if(self)
-    {
+    if (self) {
```

Comment:
> { [](start = 13, length = 2)

For braces can we adopt the rule that puts the opening '{' on a new line (the way the file was originally?) ? I am hoping that's the norm we've formalized on other platforms/languages by now? @paulcam206?

Reply:
i thought we were following apple coding style from OneNote.

---
In reply to: [300754221](https://github.com/microsoft/AdaptiveCards/pull/3146#discussion_r300754221) [](ancestors = 300754221)